python/src/opendp/context.py

```diff
@@ -723,11 +722,16 @@ def predicate(param):
 
     def __rshift__(self, other: Union[Transformation, Measurement]):
         # partials may be chained with other transformations or measurements to form a new partial
-        # TODO: Can we exercise this?
         if isinstance(other, (Transformation, Measurement)): # pragma: no cover
```

Comment:
```suggestion
        if isinstance(other, (Transformation, Measurement)):
```
ditto: Maybe covered now?